### PR TITLE
Update permissions.yml for bluebeetle

### PR DESCRIPTION
**Pull request by MrCraft888**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -4701,6 +4701,33 @@ groups:
...
+    - mortal
+    options:
+      prefix: '&bB&1l&bu&1e &bB&1e&9e&1t&ble'
+      suffix: '&1'
+    worlds:
+      survival:
+        permissions:
+        - essentials.fly
+        - essentials.speed.fly
+        - essentials.speed
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: requires proof of flying at super high speeds

Comment (MrCraft888) [inline PermissionsEx/permissions.yml]: https://www.youtube.com/watch?v=notUqx2W0qw look at the second one

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: the second one what? thats a ten minute video

Comment (MrCraft888) [inline PermissionsEx/permissions.yml]: nvm

**Pull request closed without merging**